### PR TITLE
Add new parser for ls_run_systemd_generator

### PR DESCRIPTION
**Pull request by JoySnow**
Signed-off-by: XiaoXue Wang <xiaoxwan@redhat.com>

### CODE REVIEW

File: insights/parsers/ls_run_systemd_generator.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    drwxr-xr-x.  2 0 0  60 Aug  5 07:35 nfs-server.service.d
+    drwxr-xr-x.  2 0 0 100 Aug  5 07:35 remote-fs.target.requires
+    -rw-r--r--.  1 0 0 261 Aug  5 07:35 root-mnt_nfs3.mount
+    -rw-r--r--.  1 0 0 261 Aug  5 07:35 root-mnt\x2dnfs1.mount
+    -rw-r--r--.  1 0 0 261 Aug  5 07:35 root-mnt\x2dnfs2.mount
+    drwxr-xr-x.  2 0 0  60 Aug  5 07:35 swap.target.requires
+
+Examples:
+
+    >>> ls.files_of(None) == ['boot.mount', 'boot-fake.mount', 'dev-mapper-rhel-swap.swap', '-.mount', 'root-mnt_nfs3.mount', 'root-mnt-nfs1.mount', 'root-mnt-nfs2.mount']
```
Comment (xiangce) [inline insights/parsers/ls_run_systemd_generator.py]: Use `None` as a parameter is ambiguous, the base `FileListing` have resolved this issue by parsing the `file_path`. But it needs a valid argument being passed.

See: https://github.com/RedHatInsights/insights-core/blob/9d277560c6a725266f48993e25d1c3d6b7b7ba41/insights/parsers/tests/test_ls_var_lib_mongodb.py#L18

File: insights/parsers/tests/test_ls_run_systemd_generator.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    assert ls.files_of(None) == ['boot.mount', 'boot-fake.mount', 'dev-mapper-rhel-swap.swap',
+                                '-.mount', 'root-mnt_nfs3.mount', 'root-mnt-nfs1.mount', 'root-mnt-nfs2.mount']
+    assert ls.dirs_of(None) == ['.', '..', 'local-fs.target.requires', 'nfs-server.service.d',
+                               'remote-fs.target.requires', 'swap.target.requires']
+    assert ls.listing_of(None)['-.mount'] == {'type': '-', 'perms': 'rw-r--r--.', 'links': 1, 'owner': '0', 'group': '0', 'size': 217, 'date': 'Aug  5 07:35', 'name': '-.mount', 'raw_entry': '-rw-r--r--.  1 0 0 217 Aug  5 07:35 -.mount', 'dir': None}
+
+
+def test_ls_osroot_doc_examples():
+    env = {
+        'LsRunSystemdGenerator': LsRunSystemdGenerator,
```
Comment (xiangce) [inline insights/parsers/tests/test_ls_run_systemd_generator.py]: This line is useless and can be removed.

Comment (JoySnow) [reply insights/parsers/tests/test_ls_run_systemd_generator.py]: Done. This line is removed.

File: insights/parsers/ls_run_systemd_generator.py (hunk 1 above)
Comment (JoySnow) [reply insights/parsers/ls_run_systemd_generator.py]: The real path is used instead of None.

**Pull request merged**